_data/utilities.yml

```diff
@@ -450,7 +450,7 @@ websites:
       tfa: No
 
     - name: Vodafone (IT)
-      url: http://www.vodafone.it/
+      url: https://www.vodafone.it/
```

Comment:
Link doesn't work for me due to cipher incompatibility. From looking at [ssl labs](https://www.ssllabs.com/ssltest/analyze.html?d=vodafone.it) it seems like HTTPS might not really be supported by them.

![Screen Shot 2019-07-25 at 01 41 02-fullpage](https://user-images.githubusercontent.com/3535780/61835733-544d2e00-ae7d-11e9-82ca-e34114a21972.png)